docs/content/getting-started/install.md

```diff
@@ -138,6 +138,22 @@ Check out:
 * Guides
 * Contributing
 
+# Adding folders
+
+Next, we'll add some folders into your project to give you an idea of how the
+folder system works.
+
+Remember that folders are read automatically. That way, you can drop in 
+place pre-built flakes. For example, on another project, you might have
+built a flake that configures mysql. In that project you placed it in
```

Comment:
```suggestion
place pre-built packages. For example, on another project, you might have
built a package that configures mysql. In that project you placed it in
```
Typically you would say "flake" to refer to the whole repository. And then package or "derivation" for those units of build.